db/users.js

```diff
@@ -1,25 +1,89 @@
-const client = require("./client");
-
-// database methods
-
-async function createUser({ username, password, email }) {
-  try {
-    const {
-      rows: [user],
-    } = await client.query(
-      `
-        INSERT INTO users(username, password, email)
-        VALUES($1, $2, $3)
-        ON CONFLICT (username) DO NOTHING
-        RETURNING id, username, email;
-      `,
-      [username, password, email]
-    );
-
-    return user;
-  } catch (error) {
-    throw error;
-  }
+const client = require("./client")
```

Comment:
I might refactor this to be `require('./client.js')` because Eliot agrees with the node developer that it should've been required lol